plugin/ptrace/ptracewrappers.cpp

```diff
@@ -93,7 +93,7 @@
 // FIXME:  ARM also uses sigreturn.  however, more debugging is needed before
```

Comment:
It all seems good.  No comments for this commit.

However, the commit msg is much too long (50 chars only).
And also there is a typo in the msg: `definition`, not `defanition`.